.travis.yml

```diff
@@ -58,6 +60,10 @@ install:
   - sudo su $GALAXY_TRAVIS_USER -c 'sed -i.bak "s/commands.*$/commands =/" tox.ini'
 
 script:
+  # Test submitting jobs to an external slurm cluster
+  - docker tag galaxy-docker/test quay.io/bgruening/galaxy
+  - cd $TRAVIS_BUILD_DIR/docs/slurm/test && sudo -E su $GALAXY_TRAVIS_USER -c 'bash test.sh'
+  - sudo chown -R $GALAXY_TRAVIS_USER:$GALAXY_TRAVIS_USER /tmp
```

Comment:
Yeah, have seen that. Would be great if this works it looks so much simpler :)
I'm just wondering why I did it so complicated ... hopefully there was no reason.
Thanks.